support-data-rest/src/main/java/org/uniprot/api/disease/DiseaseService.java

```diff
@@ -1,34 +1,27 @@
 package org.uniprot.api.disease;
 
-import static java.util.Collections.emptyList;
-
-import java.util.function.Supplier;
-
+import org.springframework.context.annotation.Import;
 import org.springframework.stereotype.Service;
+import org.uniprot.api.common.repository.search.QueryBoosts;
 import org.uniprot.api.rest.service.BasicSearchService;
 import org.uniprot.core.cv.disease.Disease;
-import org.uniprot.store.search.DefaultSearchHandler;
 import org.uniprot.store.search.document.disease.DiseaseDocument;
 import org.uniprot.store.search.field.UniProtSearchFields;
 
 @Service
+@Import(DiseaseQueryBoostsConfig.class)
```

Comment:
Do we need to import it?

Reply:
What are your thoughts? For annotation-based configs, then yes, @Import is a normal way of using them.